packages/salesforcedx-vscode-lightning/src/index.ts

```diff
@@ -15,6 +17,7 @@ import {
   window,
   workspace
 } from 'vscode';
+import * as vscode from 'vscode';
```

Comment:
Remove this line and just add the new dependencies to the previous vscode import.